### DISCUSSION TITLE
Replace localStorage with CookieStorage with nextjs

### DISCUSSION THREAD

**Discussion by edw19** [accounts-js/accounts#1105] [General]
I attach the Storage cookies code and it works perfectly, only there is a drawback which is that with localStorage AccountsJS it refreshed the user's session but now with Cookies Storage the session refresh is lost. I have noticed that the cookies remain stored but AccountsJS does not recognize them
![image](https://user-images.githubusercontent.com/47750859/106904760-90e02380-66c9-11eb-8973-668c15e6409b.png)

**Comment by rohit-gohri** [comment]
I think this might work https://github.com/ripeworks/local-storage-fallback rather than making our own?
